### ISSUE TITLE
Hide requests on published docs (in response to #3795)

### ISSUE THREAD

**Issue by phcoliveira** [a85/POSTMan-Chrome-Extension#5776]
@vkaegis, while I appreciate having version control on Postman, I would like to point out that it is not as straightforward as using GIT.

Pulling changes from origin seems to be some sort of accident which is recommended to comply, rather than actively choosing to pull.

You see, this can not be seen as having a master and a develop branch, because public branch (develop) is not supposed to be merged into origin (master), but quite the opposite: the public branch should pull from the origin.

This might not sound like a big deal, but consider this. Have some requests deleted on the public branch, then have some changes on other requests from origin. Before merging, Postman tell us about the recent changes on origin and advises to pull them. Not as straightforward, but it gets the job done.

Now, have some changes on the same requests which were deleted on the public branch. This time, Postman will not try to pull the changes from origin, but rather resolve the conflicts in order to merge de public branch into the origin, which is not what we want.

If we could have a dedicated button for "pull from origin", we could always get the job done, but, still, how difficult it would be to have an extra boolean property for rendering on the published docs? This should be so much easier than to implement version control management.

**Comment by vkaegis** [comment]
@phcoliveira Thanks for writing in! Pulling changes from the parent is something we're actively working on and should be out in a few weeks.

Regarding the simpler "boolean property", our general principle while working on features is to assess impact and provide solutions that can help a large community of our users. This helps us to prioritize issues as well as design solutions which can solve multiple use cases at the same time. This in turn allows us to ship these features faster so users who are blocked on certain workflows can be unblocked as quickly as possible. 

Marking this as a feature request until the Pull Changes action is available.

**Comment by phcoliveira** [comment]
@vkaegis, thank you for your readiness, but let me comment on your business rather than on your software.

Hiding requests or entire folders on the published docs is a feature that has been requested for a long time, and yet you keep saying that it will be considered or something better is yet to come.

Having version control management to filter what should or should not be displayed to the public is far from ideal because it is not the right tool for the job. Again, we, paying consumers, are telling you this, and your response to our needs is something like "no, you actually don't want that, I have something better for you".

The reason I picked Postman over Insomnia is its publishing capabilities, but I already regret paying for a full year upfront rather than monthly.

For some reason, Postman fails to acknowledge the differences among developing, consuming, documenting and releasing an API. Please have a look on this issue I wrote and, until now, was completely ignored by the core team.

postmanlabs/postman-docs#1470

**Comment by vkaegis** [comment]
Hey @phcoliveira, sorry you feel that way. :( As I mentioned, we appreciate you providing your inputs on the feature that solves your exact use case, and we continue to assess which implementation / feature is apt for this. We aren't claiming that Version Control is the ideal solution for your problem, just that it unblocks you and allows you to continue with your work. We will keep you updated if we have any more features in line with this.

**Comment by vkaegis** [comment]
@phcoliveira With reference to this: postmanlabs/postman-docs#1470

That repo is for documentation of the Postman product. Could you raise this as a separate issue in https://github.com/postmanlabs/postman-app-support? Thanks!
